src/vcHotkey.cpp

```diff
@@ -152,8 +162,8 @@ namespace vcHotkey
     }
 
     pStrings[4] = SDL_GetScancodeName((SDL_Scancode)(mappedKey & 0x1FF));
-   
-    vcStringFormat(pBuffer, bufferLen, "{0}{1}{2}{3}{4}", pStrings, 5);
+    pStrings[5] = key >= vcB_Forward && key <= vcB_Down ? ", Shift Spd++, Ctrl Spd--" : "";
```

Comment:
These strings are strange, intentional?

Reply:
I was conserving space, but probably don't need to, changed